### PR TITLE
allow only ffv.gpg for ffv apt repository

### PR DESCRIPTION
**Pull request by txt-file**
Signed-off-by: Vieno Hakkerinen <vieno@chemnitz.freifunk.net>

### CODE REVIEW

**Comment by ecsv** [comment]
If you want to use signed-by then please make sure that ffv.gpg is not in /etc/apt/trusted.gpg.d/ but in something like /etc/apt/keyrings/, /usr/share/keyrings or /usr/local/share/keyrings - otherwise ffv.gpg is still valid for all other repositories. An example can for example be found at https://vogtland.freifunk.net/apt/README

And please prefer lower case letters "signed-by" as shown in the official documentation for .list files. For .sources files, the correct version is Signed-by:

And also the comma is incorrect. It must be a space

A month ago, there was a nice post about this on planet.debian.org: https://michael-prokop.at/blog/2021/02/16/how-to-properly-use-3rd-party-debian-repository-signing-keys-with-apt/

**Review by ecsv — CHANGES_REQUESTED** [review]
See the example from https://vogtland.freifunk.net/apt/README

File: roles/common/files/etc/apt/sources.list.d/ffv.list [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: CC0-1.0
 # SPDX-FileCopyrightText: 2017-2020, Sven Eckelmann <sven@narfation.org>
 
-deb [arch=amd64] https://vogtland.freifunk.net/apt/ buster main contrib non-free
+deb [arch=amd64,Signed-By=/etc/apt/trusted.gpg.d/ffv.gpg] https://vogtland.freifunk.net/apt/ buster main contrib non-free
```
Comment (ecsv) [inline roles/common/files/etc/apt/sources.list.d/ffv.list:4]: * it has to be signed-by
* comma has to be a space
* the ffv.gpg should be moved outside of /etc/apt/trusted.gpg.d

**Comment by ecsv** [comment]
You can find the change I want to see in this PR in https://github.com/FreifunkVogtland/ansible-config/commit/apt-trusted-local

**Comment by txt-file** [comment]
https://github.com/FreifunkVogtland/ansible-config/commit/apt-trusted-local looks good to me. You already took the time to implement the requested changes so just merge https://github.com/FreifunkVogtland/ansible-config/commit/apt-trusted-local.

I learned a lot. Thanks.

**Comment by ecsv** [comment]
Merged, thanks. And also thanks for "pushing" me to finally implement the keyring stuff in the correct way on the servers :)

**Pull request closed without merging**